lib/Utils/Systemd.pm

```diff
@@ -48,10 +48,13 @@ sub disable_and_stop_service {
     $args{mask_service}   //= 0;
     $args{ignore_failure} //= 0;
 
-    systemctl("mask $service_name", ignore_failure => $args{ignore_failure}) if ($args{mask_service});
-    systemctl("disable $service_name", ignore_failure => $args{ignore_failure}) unless ($args{mask_service});
-    systemctl("stop $service_name", ignore_failure => $args{ignore_failure});
+    my $cmd = $args{mask_service} ? 'mask' : 'disable';
+    if (is_sle('<12-sp3')) {
```

Comment:
I think we should mention here why this has to be done: https://bugzilla.suse.com/show_bug.cgi?id=1135080

Reply:
I would prefer if the service is disabled before being masked.

Reply:
Running disable on masked or service being masked afterwards is useless and will fail on older SLE versions, same as mentioned bsc#1135080
Quotes from documentation:
`Using enable on masked units results in an error.`
`This is a stronger version of disable`